include/zephyr/bluetooth/mesh/brg_cfg_cli.h

```diff
@@ -145,17 +143,16 @@ int bt_mesh_brg_cfg_cli_subnet_bridge_get(uint16_t net_idx, uint16_t addr,
  *  @param net_idx Network index to encrypt the message with.
  *  @param addr    Target node address.
  *  @param val     Value to set the Subnet Bridge state to. Must be one of
- *                 @ref BT_MESH_SUBNET_BRIDGE_DISABLED or
- *                 @ref BT_MESH_SUBNET_BRIDGE_ENABLED.
+ *                 @ref BT_MESH_BRG_CFG_DISABLED or
+ *                 @ref BT_MESH_BRG_CFG_ENABLED.
  *  @param status  Status response parameter, returns one of
- *                 @ref BT_MESH_SUBNET_BRIDGE_DISABLED or
- *                 @ref BT_MESH_SUBNET_BRIDGE_ENABLED on success.
+ *                 @ref BT_MESH_BRG_CFG_DISABLED or
+ *                 @ref BT_MESH_BRG_CFG_ENABLED on success.
  *
  *  @return 0 on success, or (negative) error code on failure.
  */
-int bt_mesh_brg_cfg_cli_subnet_bridge_set(uint16_t net_idx, uint16_t addr,
-					  enum bt_mesh_subnet_bridge_state val,
-					  enum bt_mesh_subnet_bridge_state *status);
+int bt_mesh_brg_cfg_cli_set(uint16_t net_idx, uint16_t addr, enum bt_mesh_brg_cfg_state val,
```

Comment:
This is for all these functions in the client. I know the names will be a little bit long, but is it not better to keep the names closer to what is in the spec. The spec name is subnet_bridge_set, and then it is easier to connect this to the spec function. I think I would prefer to only change the brg_cfg.h names.

Reply:
There is some disagreement on this front, thoughts @PavelVPV and @alxelax?
I agree that it would be easier to connect it to the spec with the longer names, but in the other model APIs we have a tendency to go for the shorter names.

Reply:
I tend to think consistency with our other models API prevail over consistency with spec terminology simply because that makes it easier to work with the codebase. I'd probably do that as Ingar suggested from the beginning, but the pattern is already defined in other API, so my suggestion to follow it.

Reply:
I agree with @PavelVPV. I still think that the double mentioning `bridge` in one form or another doesn't add any value.

Reply:
Ok, to keep consistency with rest of the code base is more important.